### PR TITLE
[cannon] Missing vehicle function & typo fix

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
http://schteppe.github.io/cannon.js/docs/classes/RaycastVehicle.html
http://schteppe.github.io/cannon.js/docs/classes/Box.html
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/cannon/index.d.ts [hunk 1]
```diff
@@ -740,7 +741,7 @@ declare module CANNON {
 
     export class Box extends Shape {
 
-        static calculateIntertia(halfExtents: Vec3, mass: number, target: Vec3): void;
+        static calculateInertia(halfExtents: Vec3, mass: number, target: Vec3): void;
```
Comment (Gelio) [inline types/cannon/index.d.ts:744]: Nice catch there :+1:

**Review by Gelio — APPROVED** [review]
Looks good to me :smile: 

@VitorMac10 , you should probably add yourself to the `Definitions by:` list at the top of the types file, so you are recognized for this contribution and appear as reviewer for future contributions :slightly_smiling_face:

**Comment by VitorMac10** [comment]
> Looks good to me 😄
> 
> @VitorMac10 , you should probably add yourself to the `Definitions by:` list at the top of the types file, so you are recognized for this contribution and appear as reviewer for future contributions 🙂

**Comment by VitorMac10** [comment]
> Looks good to me 😄
> 
> @VitorMac10 , you should probably add yourself to the `Definitions by:` list at the top of the types file, so you are recognized for this contribution and appear as reviewer for future contributions 🙂
Thanks for the suggestion. just added my name :)

**Review by Gelio — APPROVED** [review]

**Comment by orta** [comment]
Cool, thanks!

**Pull request merged**